src/disc/omegah/OmegahConnManager.cpp

```diff
@@ -186,11 +186,10 @@ GO getMaxGlobalEntDofId(Omega_h::Mesh& mesh, Omega_h::GOs& dofGlobalIds) {
 
 std::array<Omega_h::GOs,4> OmegahConnManager::createGlobalDofNumbering() const {
   std::array<Omega_h::GOs,4> gdn;
-  GO startingOffset = 0;
+  GO offset = 0;
   for(int i=0; i<gdn.size(); i++) {
-    gdn[i] = createGlobalEntDofNumbering(mesh, i, m_dofsPerEnt[i], startingOffset);
-    const auto offset = getMaxGlobalEntDofId(mesh, gdn[i]);
-    startingOffset = offset == 0 ? startingOffset : offset;
+    gdn[i] = createGlobalEntDofNumbering(mesh, i, m_dofsPerEnt[i], offset);
+    offset += getMaxGlobalEntDofId(mesh, gdn[i])*m_dofsPerEnt[i];
```

Comment:
Ah, good catch! I will remove this counter then. To be clear, it's not "wrong", but it may make GIDs unnecessarily large, right?

Reply:
> I will remove this counter then.

Sounds good. Thank you.

> To be clear, it's not "wrong", but it may make GIDs unnecessarily large, right?

Yeah.  Exactly.

For the record, 'dofsPerEnt' is included in the global Id computation here:
https://github.com/sandialabs/Albany/blob/e13e9e07ece674c0a817de4d9fbabb8fb914b8e3/src/disc/omegah/OmegahConnManager.cpp#L171

Looking at this line:
https://github.com/sandialabs/Albany/blob/e13e9e07ece674c0a817de4d9fbabb8fb914b8e3/src/disc/omegah/OmegahConnManager.cpp#L172

Using `offset += getMaxGlobalEntDofId(...)` may also be padding the offset unnecessarily.

For example, suppose offset=100 is passed to `createGlobalEntDofNumbering(...)`, and, for this example,  creates 20 dof ids.  We then call `getMaxGlobalEntDofId(...)` and that returns 120. If we accumulated into offset then we would have 220 as the offset for the next iteration instead of 120.

Reply:
I think you're right. I think I confused dofs numbering with geo entities numbering. I will roll back these changes.